android-app-examples/SNetDemo/app/src/main/java/com/example/snetdemo/SemanticSegmentationService.java

```diff
@@ -0,0 +1,31 @@
+package com.example.snetdemo;
+
+import java.io.Closeable;
+import java.math.BigInteger;
+
+import io.singularitynet.sdk.client.FixedPaymentChannelPaymentStrategy;
+import io.singularitynet.sdk.client.PaymentStrategy;
+import io.singularitynet.sdk.client.ServiceClient;
+import io.singularitynet.service.semanticsegmentation.SemanticSegmentationGrpc;
+
+public class SemanticSegmentationService implements Closeable
```

Comment:
I would suggest move this code into `ImageSegmentationActivity` `onCreate()` and `onDestroy()` methods.

Reply:
Done